docs/rpc-spec.md

```diff
@@ -344,7 +344,7 @@ Files are returned in the order they are laid out in the torrent. References to
 | `fromTracker`  | number     | tr_stat
 
 
-`pieces`: A bitfield holding pieceCount flags which are set to 'true' if we have the piece matching that position. JSON doesn't allow raw binary data, so this is a base64-encoded string. (Source: tr_torrent)
+`pieces`: A bitfield holding pieceCount flags which are set to 'true' if we have the piece matching that position. JSON doesn't allow raw binary data, so this is a base64-encoded string. Pieces are guaranteed to be written on disk if sequential download is enabled. Otherwise, data might still be in cache only. (Source: tr_torrent)
```

Comment:
I prefer this new description to be added to `cache-size-mb` instead.

Reply:
The RPC docs looks good now. Would you please copy this new sentence to here as well?

https://github.com/transmission/transmission/blob/ac5c9e082da257e102eb4ff18f2e433976a585d1/docs/Editing-Configuration-Files.md?plain=1#L76